plugins/module_utils/healthchecksio.py

```diff
@@ -117,6 +123,19 @@ def healthchecksio_argument_spec():
                 required=True,
                 no_log=True,
             ),
+            api_base_url=dict(
```

Comment:
You'll want to run [black](https://github.com/psf/black) on this file to get the test to pass.